pkg/volume/util/fsquota/quota_linux.go

```diff
@@ -439,3 +445,24 @@ func ClearQuota(m mount.Interface, path string) error {
 	}
 	return nil
 }
+
+// RecordProjectIDOnProjectFiles -- Records project file for the given directory-path to /etc/projects and /etc/projectid
+func RecordProjectIDOnProjectFiles(podVolumeDir string) error {
+	if id, ok := getQuotaIDFromDirPath(podVolumeDir); ok {
+		fProjects, fProjid, err := openAndLockProjectFiles()
```

Comment:
You should be able to use `createProjectID` for this.

Reply:
Done, also please check the comment above